src/applications/vaos/tests/new-appointment/redux/helpers/formSubmitTransformers.unit.spec.js

```diff
@@ -17,7 +17,7 @@ describe('VAOS data transformation', () => {
             morning: true,
           },
           email: 'test@va.gov',
-          visitType: 'office',
+          visitType: 'clinic',
```

Comment:
Our visit type ids lined up well for phone and telehealth, but we had office instead of clinic. Switching means that we can set kind directly from the `visitType` field in the form.